src/routes/home.tsx

```diff
@@ -5,27 +5,55 @@ import Spinner from '@/components/spinner'
 import { compareEvents } from '@/utils/compare-events'
 import { useGeoLocation } from '@/utils/use-geo-location'
 import { useEvents } from '@/cache/events/use'
+import { useState } from 'preact/hooks'
 
 const now = new Date()
-
 const Home = () => {
   const events = useEvents()
   const location = useGeoLocation()
+  const [term = '', setTerm] = useState(0);
+
+  function searchingFor(term: string | number) {
```

Comment:
I think it would make sense for this to be always passed a string.

```suggestion
  function searchingFor(term: string) {
```